PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/mysql/MySQLPropertyView.java

```diff
@@ -0,0 +1,268 @@
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.microsoft.intellij.helpers.mysql;
+
+import com.microsoft.azure.management.mysql.v2020_01_01.Server;
+import com.microsoft.azure.management.mysql.v2020_01_01.ServerState;
+import com.microsoft.azure.management.mysql.v2020_01_01.implementation.DatabaseInner;
+import com.microsoft.azure.management.mysql.v2020_01_01.implementation.FirewallRuleInner;
+import com.microsoft.azure.management.resources.Subscription;
+import com.microsoft.azure.toolkit.intellij.common.AzureHideableTitledSeparator;
+import com.microsoft.azure.toolkit.intellij.mysql.ConnectionSecurityPanel;
+import com.microsoft.azure.toolkit.intellij.mysql.ConnectionStringsOutputPanel;
+import com.microsoft.azure.toolkit.intellij.mysql.DatabaseComboBox;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTask;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
+import com.microsoft.azuretools.azurecommons.util.Utils;
+import com.microsoft.azuretools.core.mvp.model.AzureMvpModel;
+import com.microsoft.azuretools.core.mvp.model.mysql.MySQLMvpModel;
+import com.microsoft.intellij.helpers.base.BaseEditor;
+import com.microsoft.tooling.msservices.serviceexplorer.azure.mysql.MySQLModule;
+import com.microsoft.tooling.msservices.serviceexplorer.azure.mysql.MySQLProperty;
+import com.microsoft.tooling.msservices.serviceexplorer.azure.mysql.MySQLPropertyMvpView;
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ItemEvent;
+import java.util.List;
+
+public class MySQLPropertyView extends BaseEditor implements MySQLPropertyMvpView {
+
+    public static final String ID = "com.microsoft.intellij.helpers.mysql.MySQLPropertyView";
+
+    private AzureHideableTitledSeparator overviewSeparator;
+    private MySQLPropertryOverviewPanel overview;
+    private AzureHideableTitledSeparator connectionSecuritySeparator;
+    private ConnectionSecurityPanel connectionSecurity;
+    private AzureHideableTitledSeparator connectionStringsSeparator;
+    private ConnectionStringsOutputPanel connectionStringsJDBC;
+    private ConnectionStringsOutputPanel connectionStringsSpring;
+    private JPanel rootPanel;
+    private JPanel contextPanel;
+    private JScrollPane scrollPane;
+    private MySQLPropertyActionPanel propertyActionPanel;
+    private DatabaseComboBox databaseComboBox;
+    private JLabel databaseLabel;
+    public static final String MYSQL_OUTPUT_TEXT_PATTERN_SPRING =
+            "spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver" + StringUtils.LF +
```

Comment:
what is the difference from 
https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/SystemUtils.html#LINE_SEPARATOR